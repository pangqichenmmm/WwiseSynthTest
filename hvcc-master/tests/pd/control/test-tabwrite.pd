#N canvas 750 181 444 399 10;
#N canvas 0 22 450 278 (subpatch) 0;
#X array array1 3 float 2;
#X coords 0 1 3 -1 200 140 1 0 0;
#X restore 192 243 graph;
#X obj 24 258 tabread array1;
#X obj 21 8 loadbang;
#X obj 24 280 print;
#X msg 85 217 0;
#X msg 55 217 1;
#X msg 25 217 2;
#X obj 22 181 t b b b;
#X obj 287 89 t b b;
#X msg 290 128 2;
#X msg 322 128 0;
#X obj 217 89 t b b;
#X msg 252 128 1;
#X msg 220 128 120;
#X obj 153 93 t b b;
#X obj 83 93 t b b;
#X msg 188 132 1;
#X msg 153 132 -100;
#X msg 115 132 2;
#X msg 83 132 1;
#X obj 98 184 tabwrite array1;
#X obj 21 30 t b b b;
#X obj 84 63 t b b b b;
#X connect 1 0 3 0;
#X connect 2 0 21 0;
#X connect 4 0 1 0;
#X connect 5 0 1 0;
#X connect 6 0 1 0;
#X connect 7 0 6 0;
#X connect 7 1 5 0;
#X connect 7 2 4 0;
#X connect 8 0 9 0;
#X connect 8 1 10 0;
#X connect 9 0 20 0;
#X connect 10 0 20 1;
#X connect 11 0 13 0;
#X connect 11 1 12 0;
#X connect 12 0 20 1;
#X connect 13 0 20 0;
#X connect 14 0 17 0;
#X connect 14 1 16 0;
#X connect 15 0 19 0;
#X connect 15 1 18 0;
#X connect 16 0 20 1;
#X connect 17 0 20 0;
#X connect 18 0 20 1;
#X connect 19 0 20 0;
#X connect 21 0 7 0;
#X connect 21 1 22 0;
#X connect 21 2 7 0;
#X connect 22 0 15 0;
#X connect 22 1 14 0;
#X connect 22 2 11 0;
#X connect 22 3 8 0;
