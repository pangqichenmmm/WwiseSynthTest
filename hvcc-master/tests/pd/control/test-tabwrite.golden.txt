[@ 0.000] print: 0
[@ 0.000] print: 0
[@ 0.000] print: 0
[@ 0.000] print: 2
[@ 0.000] print: -100
[@ 0.000] print: 1
